### PR TITLE
EWPP-3305: Fix date translations and remove duplicates.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

File: translations/oe_content-el.po [hunk 1]
```diff
@@ -189,45 +189,3 @@ msgstr "Παρασκευή"
 
 msgid "Saturday"
 msgstr "Σάββατο"
-
```
Comment (22Alexandra) [inline translations/oe_content-el.po:192]: All long month names need to be updated for EL.

File: translations/oe_content-fi.po [hunk 2]
```diff
@@ -189,45 +189,3 @@ msgstr "perjantai"
 
 msgid "Saturday"
 msgstr "lauantai"
-
```
Comment (22Alexandra) [inline translations/oe_content-fi.po:192]: All long month names need to be updated for FI.

File: translations/oe_content-hr.po [hunk 3]
```diff
@@ -189,45 +189,3 @@ msgstr "petak"
 
 msgid "Saturday"
 msgstr "subota"
-
```
Comment (22Alexandra) [inline translations/oe_content-hr.po:192]: Please update the long month names.

File: translations/oe_content-da.po [hunk 4]
```diff
@@ -3,231 +3,189 @@ msgstr ""
...
+msgstr "november"
 
 msgctxt "Long month name"
 msgid "December"
-msgstr "December"
+msgstr "december"
 
 msgid "Jan"
-msgstr "Jan."
+msgstr "jan."
```
Comment (22Alexandra) [inline translations/oe_content-da.po]: Why this dot? I might misunderstand the excel sheet but the dot is after the day, not after the month:
DA - short month name

I think we might need to remove the dot from every month name.

File: translations/oe_content-ro.po [hunk 5]
```diff
@@ -189,45 +189,3 @@ msgstr "vineri"
 
 msgid "Saturday"
 msgstr "sâmbătă"
-
```
Comment (22Alexandra) [inline translations/oe_content-ro.po:192]: For RO I don't see any dot after the short month name, I think we need to remove it (seems outdated).

**Review by 22Alexandra — CHANGES_REQUESTED** [review]

File: translations/oe_content-da.po (hunk 4 above)
Comment (nagyad) [reply translations/oe_content-da.po]: But what if that is their way to make the abbreviation?

Comment (22Alexandra) [reply translations/oe_content-da.po]: They use %B for the long month name and %b for short month name. Seems good now, thanks.

**Review by 22Alexandra — APPROVED** [review]

**Review by sergepavle — APPROVED** [review]

**Pull request merged**